code/game/machinery/doors/brigdoors.dm

```diff
@@ -211,7 +211,7 @@
 
 	for(var/obj/machinery/treadmill_monitor/T in targets)
 		T.total_joules = 0
-		T.on = 1
+		T.on = TRUE
 
 	return 1
```

Comment:
```suggestion
	return TRUE
```
I'm assuming these are supposed to be bools as well. All instances of this proc being used don't actually check the value returned so there's not really much point, but code readability.

Reply:
Yeah, I wasn't going to go there, as that's another big can of worms that I know once I started in one spot I'd want to keep doing everywhere.... but just accepting your suggestions, I can probably do that with out feeling like I need to do a find in files for 'return 1'. 😛